upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -335,13 +345,26 @@ func (e *InstanceTemplate) mapToGCE(project string, region string) (*compute.Ins
 		})
 	}
 
+	var accelerators []*compute.AcceleratorConfig
+	if len(e.GuestAccelerators) > 0 {
```

Comment:
What happens when you remove accelerator from config? With this here it won't apply any change.